### ISSUE TITLE
Allow updating images in Gallery component 

### ISSUE THREAD

**Issue by numediaweb** [akveo/kittenTricks#75]
### Issue type

**I'm submitting a ...**  

* [x] feature request

### Issue description

**Current behavior:**
The gallery images can't be updated once rendered:
```
    renderGallery = () => {
        return (
            <Gallery items={this.state.images}/>
        );
    };
```

**Expected behavior:**
The gallery should update images if the state of items changes

**Steps to reproduce:**
Include the gallery componenet with some images (see example in [kittenTricks app](https://github.com/akveo/kittenTricks/blob/master/app/screens/social/profile1.js#L59)) the update the items array => gallery won't update.

**Related code:**
This is how I fixed it by modifing components/gallery.js
```
    componentWillReceiveProps(nextProps) {
        this.setState({ data: nextProps.items });
    }
```

**Comment by artyorsh** [comment]
Hi @numediaweb 
Thanks for report. Feel free to open pull request with your changes
Also: have you seen [RkGallery](https://akveo.github.io/react-native-ui-kitten/#/docs/ui-components/rkgallery)?

**Comment by numediaweb** [comment]
@artyorsh PR created: https://github.com/akveo/kittenTricks/pull/76

**Comment by artyorsh** [comment]
Thanks for your interest @numediaweb 
We've released v2.0. Check this out :fire:
